simulation/types.go

```diff
@@ -1,7 +1,14 @@
 package simulation
 
-import "io"
+import (
+	"github.com/openthread/ot-ns/types"
```

Comment:
Needs to use `goimports` to order the imported modules. @EskoDijk

Reply:
Fixed in an earlier commit. Sometimes the issue can come back to the automatic IDE actions.